docs/source/notebooks/GP-Circular.ipynb

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
<ul><li>I don't think this plot is understable at first sight (I had to put a lot of thought into it to get what it means). Some axis labels and / or text explanation could help?</li><li>Also you can get rid of matplotlib object printing by adding a ";" at the end of the plot</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4082/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Looks nice now, thanks! As a final touch, do you think you can add axis labels, to speed-up understanding by readers?</p>

Reply:
<p>added labels, they are explainable now</p>